shardingsphere-governance/shardingsphere-governance-core/shardingsphere-governance-core-config/src/test/resources/yaml/metadata.yaml

```diff
@@ -30,15 +30,4 @@ configuredSchemaMetaData:
             name: PRIMARY          
 unconfiguredSchemaMetaDataMap:
   ds_0:
-    tables:
-      t_user:
-        columns:
-          id:
-            caseSensitive: false
-            dataType: 0
-            generated: false
-            name: id
-            primaryKey: true
-        indexes:
-          primary:
-            name: PRIMARY
+    - t_user
```

Comment:
Please add a blank line under this line.

Reply:
done